### DISCUSSION TITLE
Not able to login from google

### DISCUSSION THREAD

**Discussion by nerdyk3** [outline/outline#5247] [Self Hosting]
I added all the detail required for Google authentication
but after that too I am not able to log in from Google.
attached screenshot
<img width="1064" alt="Screenshot 2023-04-24 at 11 48 30 AM" src="https://user-images.githubusercontent.com/22032094/233914673-03a9e0f1-9001-4d4b-9e14-7e8e0797b96a.png">

**Comment by bmm-alc** [comment]
impossible to tell what's going on. Did you look in the logs.

**Comment by nerdyk3** [comment]
From where I get the logs, I tried to find them everywhere.
I deployed it on Linux machine

On Mon, Apr 24, 2023 at 1:19 PM Baptiste Mille-Mathias <
***@***.***> wrote:

> impossible to tell what's going on. You need to look in the logs.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/outline/outline/discussions/5247#discussioncomment-5705017>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFIC5XXKBAJZQX3SEWIBQDTXCYWCJANCNFSM6AAAAAAXJDISP4>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Thank You*
*KUNAL KUMAR KUSHWAHA*
